modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/display/context/FDSSampleDisplayContext.java

```diff
@@ -62,11 +62,19 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 		return Arrays.asList(
 			new FDSActionDropdownItem(
 				null, "view", "sampleMessage",
-				LanguageUtil.get(_fdsRequestHelper.getRequest(), "view"), null,
+				LanguageUtil.get(_fdsRequestHelper.getRequest(), "Sample View"), null,
```

Comment:
`Sample View` should be a lang key in portal lang, I'll add them to my [PR here](https://github.com/liferay-frontend/liferay-portal/pull/2262), ok?

Reply:
oh got it, thanks!